src/modules/form/types.ts

```diff
@@ -83,6 +82,8 @@ export interface DynamicInputCommonProps {
   placeholder?: string;
   breakpoints?: BreakpointOverrides;
   ariaLabelledBy?: string;
+  maxWidth?: number | string;
+  inputWidth?: number;
```

Comment:
Comments here would be helpful! Is my understanding correct?

```suggestion
  maxWidth?: number | string; // maximum width of form element (including the label)
  inputWidth?: number; // fixed with of input element (does not affect label)
```

Reply:
Yep exactly!